configure

```diff
@@ -11,6 +11,7 @@ try() {
     fi
 }
 
+try python3 "$@"
```

Comment:
I think this is the source of breakage on MinGW.
MSYS2 installs Cygwin like versions of Python 2 and 3 which handle paths differently and that does not work with typical native Windows programs.
To solve that Rust adds native Python 2 installed on CI in this line to the PATH: https://github.com/rust-lang/rust/blob/a44774c3a9739b2eea8923e09d67b14312c78ef3/src/ci/scripts/install-msys2-packages.sh#L16

The solution here would be to either install `mingw-w64-${ARCH}-python` package or add Python 3 from CI (if available).

Reply:
I thought I did install it for windows targets but maybe I forgot one. Thanks for the feedback!

Reply:
If you are talking about https://github.com/rust-lang/rust/pull/66605/files#diff-41e6c29c6b4c3c1ce121c5f1c01b6f1bR55 it's not ran on the CI because Rust defines `CUSTOM_MINGW=1`.
You might want to contact infra about preferred solution but I suppose there is Python 3 already available somewhere on the CI.

Reply:
I suspect if we do not have a python 3 in CI we can likely add it, but we would need a concrete proposal (ideally a PR) that does so, if it needs some S3 command run then we can do that.